chrome-extension/content.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Content script for all urls
+ */
+
+//Insert bulma stylesheet
+//TODO: STOP USING BULMA ONCE I WRITE MY OWN CSS FOR ALL EXTENSION ELEMENTS
+const bulmaLink = document.createElement('link');
+bulmaLink.setAttribute(
+  'href',
+  'https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css'
+);
+bulmaLink.setAttribute('rel', 'stylesheet');
+document.head.appendChild(bulmaLink);
+
+//Inject invisible sidebar onto page
+const sidebar = document.createElement('div');
+sidebar.id = 'lobstericecream';
+sidebar.classList.add('panel', 'injection-panel', 'px-4', 'py-4');
+document.body.appendChild(sidebar);
+
+//Inject invisible highlight icon onto page
+const iconButton = document.createElement('button');
+iconButton.classList.add('injection-icon');
+iconButton.innerHTML = 'Save';
+//TODO: CHANGE BUTTON TO AN ACTUAL ICON!!
+
+document.body.appendChild(iconButton);
+
+//Keep track of last used highlight
+let lastContent;
+
+//Show extension icon when selecting text on page
+document.onclick = (event) => {
+  if (iconButton.style.display === 'block') {
+    iconButton.style.display = 'none';
+  }
+  const content = window.getSelection().toString();
+  //TODO: Trim Text
+  if (content && content !== lastContent) {
+    showIcon(event, content);
+    lastContent = content;
+    //TODO: Keeping for now for future reference:
+    //chrome.storage.sync.set({ content });
```

Comment:
Consider removing this. Commenting out code is generally not necessary because of version control

Reply:
Very True, will do!